applicationLibrary/molecularDynamics/tests/singleSiteTests/ForceCalculationTest.cpp

```diff
@@ -11,10 +11,12 @@
 #include "molecularDynamicsLibrary/LJFunctor.h"
 #include "testingHelpers/commonTypedefs.h"
 
+using forceCalculationTestLJFunctor =
+    LJFunctorType</* shifting */ false, /*mixing*/ false, autopas::FunctorN3Modes::Both, /*globals*/ false,
+                  /*relevantForTuning*/ true>;
```

Comment:
```suggestion
using ForceCalculationTestLJFunctor =
    LJFunctorType</* shifting */ false, /*mixing*/ false, autopas::FunctorN3Modes::Both, /*globals*/ false,
                  /*relevantForTuning*/ true>;
```
Type -> Capital letter

Name is a bit unwieldy but ok...

Reply:
Done in 7fa81126